runtime/jvmti/jvmtiHook.c

```diff
@@ -337,6 +341,63 @@ jvmtiHookMethodEnter(J9HookInterface** hook, UDATA eventNum, void* eventData, vo
 	TRACE_JVMTI_EVENT_RETURN(jvmtiHookMethodEnter);
 }
 
+#if JAVA_SPEC_VERSION >= 19
+static void
+jvmtiHookVirtualThreadStarted(J9HookInterface** hook, UDATA eventNum, void* eventData, void* userData)
+{
+	J9VirtualThreadStartedEvent * data = eventData;
+	J9JVMTIEnv * j9env = userData;
+	jvmtiEventThreadStart callback = j9env->callbacks.VirtualThreadStart;
+	J9VMThread * currentThread = data->currentThread;
+
+	Trc_JVMTI_jvmtiHookVirtualThreadStarted_Entry();
+
+	ENSURE_EVENT_PHASE_START_OR_LIVE(jvmtiHookVirtualThreadStarted, j9env);
+
+	/* Call the event callback */
+
+	if (callback != NULL) {
+		jthread threadRef;
+		UDATA hadVMAccess;
+		UDATA javaOffloadOldState = 0;
+
+		if (prepareForEvent(j9env, currentThread, currentThread, JVMTI_EVENT_VIRTUAL_THREAD_START, &threadRef, &hadVMAccess, FALSE, 0, &javaOffloadOldState)) {
```

Comment:
code-lines more than 100 chars should be spread across multiple lines. this should be applied to all places where code-line exceeds 100 chars.
```suggestion
		if (prepareForEvent(
				j9env, currentThread, currentThread, JVMTI_EVENT_VIRTUAL_THREAD_START,
				&threadRef, &hadVMAccess, FALSE, 0, &javaOffloadOldState)
		) {
```